src/Header/RequestHandler.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "./OperateData.h"
```

Comment:
Please add include path in cmake, so you don't have to add "./" to your include directive.

Reply:
Done